webpack/scenes/AlternateContentSources/Create/Steps/ACSReview.js

```diff
@@ -0,0 +1,104 @@
+import React, { useContext } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  TextContent,
+  TextList,
+  TextListVariants,
+  TextListItem,
+  TextListItemVariants,
+} from '@patternfly/react-core';
+import ACSCreateContext from '../ACSCreateContext';
+import WizardHeader from '../../../ContentViews/components/WizardHeader';
+
+const ACSReview = () => {
+  const {
+    name, description, acsType, contentType,
+    smartProxies, url, subpaths, verifySSL,
+    authentication, sslCertName, sslKeyName, username,
+    password, caCertName,
+  } = useContext(ACSCreateContext);
+
+  return (
+    <>
+      <WizardHeader
+        title={__('Review Details')}
+        description={__('Review the information below and click Add to add your source. ' +
+          'To edit details in previous steps, click Back or any step on the left')}
```

Comment:
We need a period at the end to be consistent with the "URL and paths" part of the wizard.

Reply:
Should the other description texts have periods too? Not sure what our standard should be.